mud/lp-245/obj/trace.c

```diff
@@ -386,7 +386,7 @@ int Goto(string str) {
 /*
  * This will not work because command() only works for command_giver.
  */
-int in(string str) {
+int do_in(string str) {
```

Comment:
note to reviewers: I wasn't able to find any callers of the originally named `in` function from `mud/lp-245/obj/trace.c` or `mud/lp-245/obj/trace2.c` to fix for this adjustment :thinking:

Reply:
If there is no usage of this function, then imho the proper fix would be to remove it altogether.

Reply:
Removed both in 2ad735e